javascript/javascript1/week3/seriersDuration.js

```diff
@@ -0,0 +1,52 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+function logOutSeriesText() {
+  const ageInHours = 80 * 365 * 24;
+  for (i = 0; i < seriesDurations.length; i++) {
+    let title = seriesDurations[i]["title"];
+    //console.log(title);
+    let daysPropInHours = seriesDurations[i]["days"] * 24;
+    //console.log(daysPropInHours);
+
+    let minutesInHour = seriesDurations[i]["minutes"] / 60;
+
+    let percentageOfAnAverageLifespan =
+      ((daysPropInHours + minutesInHour + seriesDurations[i]["hours"]) /
+        ageInHours) *
+      100;
+
+    console.log(
+      title + " " + percentageOfAnAverageLifespan.toFixed(3) + "% of my life"
+    );
+
+    seriesDurations[i].percentage = percentageOfAnAverageLifespan;
```

Comment:
This is fine. Other suggestion is to convert lifespan from `80` years to minutes, then inside the loop calculate each series duration in minutes like this
```js
  function logOutSeriesText() {
    const lifespan = 80 * 365 * 24 * 60; // minutes
    let total = 0;
    for (let i = 0; i < seriesDurations.length; i++) {
      const seriesDuration = seriesDurations[i].days * 24 * 60 + seriesDurations[i].hours * 60;
      const percent = (seriesDuration / lifespan) * 100;
      total += percent;
      console.log(`${seriesDurations[i].title} took ${percent}% of my life`);
    }
    console.log(`In total that is ${total}% of my life`);
  }
``` 
As you can see for total, it is easy to get it by adding each percent at for loop.